pinot-broker/src/main/java/org/apache/pinot/broker/queryquota/HelixExternalViewBasedQueryQuotaManager.java

```diff
@@ -46,6 +55,8 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import static org.apache.pinot.spi.utils.CommonConstants.Helix.DATABASE_QUERY_RATE_LIMIT;
+
 
 /**
  * This class is to support the qps quota feature.
```

Comment:
nit: this Javadoc should probably be updated - currently it states that `it only gets updated when a new table added or a broker restarted`.